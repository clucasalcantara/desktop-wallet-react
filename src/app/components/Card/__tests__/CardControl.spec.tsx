import React from "react";
import { render } from "@testing-library/react";

import { CardControl } from "../";

describe("Card Control", () => {
	it("should render", () => {
		const { container, asFragment } = render(<CardControl />);

		expect(container).toBeTruthy();
		expect(asFragment()).toMatchSnapshot();
	});
});
