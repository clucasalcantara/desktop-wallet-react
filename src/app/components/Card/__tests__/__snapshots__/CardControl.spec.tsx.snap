// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Control should render 1`] = `
<DocumentFragment>
  <label
    class="CardControl___StyledLabel-sc-28luv0-4 gNrkXm"
  >
    <input
      class="CardControl__Input-sc-28luv0-0 lbnjeC"
      type="checkbox"
      value=""
    />
    <div
      class="sc-AxjAm CardControl__CustomCard-sc-28luv0-1 kWIYfA"
    />
  </label>
</DocumentFragment>
`;
